pom.xml

```diff
@@ -225,7 +225,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
+            <version>4.5.13</version>
```

Comment:
how did you find the version that we should be using?

Reply:
Few ways. 
1. Latest version in maven. 
2. JDBC also bumped up to this version. 
3. Succesful test on PR: https://travis-ci.org/github/snowflakedb/snowflake-ingest-java/builds/744319475

Reply:
Checked with whitesource software. version 4.5.13 fixes the mentioned vulnerability and is approved by it.

Reply:
@sfc-gh-giyer can I push this?